projects/optic/src/config.ts

```diff
@@ -35,6 +35,11 @@ const DefaultOpticCliConfig: OpticCliConfig = {
 };
 
 const CaptureConfigData = Type.Object({
+  config: Type.Optional(
+    Type.Object({
+      request_concurrency: Type.Number(),
```

Comment:
Think request_concurrency should be optional as well (if we add more config) - 
```suggestion
      request_concurrency: Type.Optional(Type.Number()),
```